app/controllers/citizens/additional_accounts_controller.rb

```diff
@@ -1,6 +1,9 @@
 module Citizens
   class AdditionalAccountsController < CitizenBaseController
-    def index; end
+    def index
+      legal_aid_application.update!(has_offline_accounts: nil)
+      legal_aid_application.update!(state: 'provider_submitted')
```

Comment:
WE shouldn't be changing the state like this - we should use a transition in the state machine to do it, that way we can keep control of what state changes are legal.

You may need to create a new transition inside the `provider_submitted` event